.github/workflows/prepare_review_stage.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Set up Python 3.10
         uses: actions/setup-python@v5
         with:
-          python-version: 3.10
+          python-version: '3.10'
```

Comment:
If action `setup-python@v5` needs a properly quoted string for the `python-version` argument, then I suppose that other workflows need the same fix:
- welcome
- documentation-deploy (but this uses v4 instead of v5...)
- prepare_review_stage
- comment_bot
- deploy
- review_bot